homeassistant/components/statistics/sensor.py

```diff
@@ -1,25 +1,25 @@
 """Support for statistics for sensor values."""
+from collections import deque
 import logging
 import statistics
-from collections import deque
 
 import voluptuous as vol
 
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.recorder.util import execute, session_scope
```

Comment:
We can't have this at the top either. The util module has sqlalchemy imports at the top.

Reply:
So where should I move this import?

Reply:
To line 280.

Reply:
Okay now I understand what you want to accomplish.

Reply:
Done